components/typography/styles.js

```diff
@@ -45,11 +65,26 @@ export const heading1Styles = css`
 		line-height: 2.4rem;
 		margin: 1.5rem 0 1.5rem 0;
 	}
+	:host([skeleton]) .d2l-heading-1.d2l-skeletize {
+		height: 2.4rem;
+		overflow: hidden;
+	}
+	:host([skeleton]) .d2l-heading-1.d2l-skeletize::before {
+		top: 0.45rem;
+		bottom: 0.45rem;
+	}
 	@media (max-width: 615px) {
 		.d2l-heading-1 {
 			font-size: 1.5rem;
 			line-height: 1.8rem;
 		}
+		:host([skeleton]) .d2l-heading-1.d2l-skeletize {
+			height: 1.8rem;
```

Comment:
The fixed heights on headings in skeleton mode is to handle that case where design would like multi-line headings to show up as a single line. So we set the height equal to the line height.